ccd-definition/AuthorisationCaseField/CareSupervision/gatekeeper.json

```diff
@@ -2287,5 +2287,40 @@
     "CaseFieldID": "draftOrder10Document",
     "UserRole": "caseworker-publiclaw-gatekeeper",
     "CRUD": "CRU"
+  },
+  {
+    "LiveFrom": "01/01/2017",
```

Comment:
most of these fields seems to be temporary fields that are not saved in case data. If gatekeeper cannot start event then there is no need to grant access to these fields

Reply:
Removed temporary fields authorisation entries for gatekeeper, magistrate, judiciary and barristor roles